build.sh

```diff
@@ -50,4 +71,41 @@ build_file_list() {
   find _common -type f ! -name README.md ! -path '_common/assets/*'  -printf '%P\n' >> .bootstrap-registry
 }
 
-build_file_list
\ No newline at end of file
+builder_run_action        build-files:shared-sites     build_file_list
+
+# Site actions
+
+site_clone() {
+  local site=$1
+  cd "$THIS_SCRIPT_PATH/.."
+  if [[ ! -d $site ]]; then
+    git clone https://github.com/keymanapp/$site
+  fi
+}
+
+site_pull() {
+  local site=$1
+  cd "$THIS_SCRIPT_PATH/../$site"
+  # note: this will die if there are changes that prevent branch switch
+  git switch master
```

Comment:
Would we ever want to spool up staging sites?

(though website-local-proxy doesn't have one)